### ISSUE TITLE
Metro Exodus EE - no GUI

### ISSUE THREAD

**Issue by donizettilorenzo** [cdozdil/OptiScaler#6]
XeSS is working but no GUI!
And in some games (Crysis 2 Remastered) the GUI is affected by the game.

Maybe you can use the ReShade solutions for a more robust in-game GUI?

**Comment by cdozdil** [comment]
After final release of 0.5.0 İ am planning to work on this topic 👍🏻

**Comment by cdozdil** [comment]
Could you try this build, it's still a bit experimental but probably will be ok.

https://github.com/cdozdil/OptiScaler/actions/runs/9066919340

**Comment by donizettilorenzo** [comment]
Issue still present, no menu GUI!

**Comment by cdozdil** [comment]
Yes, I have reproduced the issue but with Metro Exodus EE i don't receive any keypresses etc. Probably using a different aproach. 

I will check it again when i have some free time.

**Comment by DrBob2** [comment]
It still doesn't render the GUI with the latest v0.6.5-pre4, Scaling works fine though so not really a problem unless wanting to switch on the fly.

**Comment by MaxMo20** [comment]
Same Issue in multiple games I play. I have posted an issue to see if it on my side. Probably is.
